### PR TITLE
Update requirements to latest versions

### PR DESCRIPTION
**Pull request by Ian-Foote**
* Fix allowed version ranges in `setup.py`.

### CODE REVIEW

**Comment by Ian-Foote** [comment]
@meshy Review?

File: setup.py [hunk 1]
```diff
@@ -7,8 +7,8 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        'FeinCMS>=1.7',
-        'django-imagekit==3.2.1',
+        'FeinCMS>=1.7,<2',
```
Comment (meshy) [inline setup.py]: As far as I am aware, FeinCMS does not follow semver.

**Comment by meshy** [comment]
Looks ok to me bar the feincms version check.

File: setup.py (hunk 1 above)
Comment (Ian-Foote) [inline setup.py]: You think it should be `>=1.7,<1.11`?

Comment (meshy) [inline setup.py]: Given that django 1.7 is only supported in FeinCMS >= 1.10, I think we should be going for `>=1.10,<1.11`

**Comment by meshy** [comment]
probably want to remove this https://github.com/incuna/incuna-videos/blob/v4.0.0/videos/tests/run.py#L41-L44

**Comment by Ian-Foote** [comment]
Updated.

**Comment by meshy** [comment]
:+1:

**Comment by Ian-Foote** [comment]
@meshy I'm thinking I'll release this as `v4.0.1`. Any objection?

**Comment by meshy** [comment]
Dropping support for a version of django? Would go for Major version personally...

**Comment by Ian-Foote** [comment]
Ok.

**Pull request merged**